app/i18n/locales/zh_TW.js

```diff
@@ -0,0 +1,662 @@
+	Choose_file: '選擇檔案',
+	Choose_where_you_want_links_be_opened: 'Choose where you want links be opened',
+	Code: '程式碼',
+	Code_or_password_invalid: '驗證碼或密碼不正確',
+	Collaborative: '協作',
+	Confirm: '確認',
+	Connect: '連接',
+	Connected: '已連接',
+	connecting_server: '連線至伺服器',
+	Connecting: '連接中',
+	Contact_us: '聯絡我們',
+	Contact_your_server_admin: '請聯絡系統管理員',
+	Continue_with: '繼續採用',
+	Copied_to_clipboard: '複製到剪貼簿',
+	Copy: '複製',
+	Conversation: '對話',
+	Permalink: '永久連結',
+	Certificate_password: '憑證密碼',
+	Clear_cache: '清除本機資料',
+	Clear_cache_loading: '清除快取',
+	Whats_the_password_for_your_certificate: '您的證書密碼是?',
+	Create_account: '新建帳戶',
+	Create_Channel: '新建頻道',
+	Create_Direct_Messages: '新增私訊',
+	Create_Discussion: '新增論壇',
+	Created_snippet: '新增程式碼片段',
+	Create_a_new_workspace: '建立一個新的工作區',
+	Create: '建立',
+	Custom_Status: '自訂狀態',
+	Dark: '深色',
+	Dark_level: '深色程度',
+	Default: '預設',
+	Default_browser: '預設瀏覽器',
+	Delete_Room_Warning: '刪除一個房間將連帶刪除房間內的所有訊息。這是不可逆的操作。',
+	Department: '部門',
+	delete: '刪除',
+	Delete: '刪除',
+	DELETE: '刪除',
+	deleting_room: '正在刪除房間',
+	description: '描述',
+	Description: '描述',
+	DESKTOP_OPTIONS: 'DESKTOP OPTIONS',
+	DESKTOP_NOTIFICATIONS: 'DESKTOP NOTIFICATIONS',
+	Desktop_Alert_info: '這些通知將發送至桌面',
+	Directory: '目錄',
+	Direct_Messages: '私訊',
+	Disable_notifications: '禁用訊息通知',
+	Discussions: '論壇',
+	Discussion_Desc: 'Help keeping an overview about what\'s going on! By creating a discussion, a sub-channel of the one you selected is created and both are linked.',
+	Discussion_name: '論壇名稱',
+	Done: '完畢',
+	Dont_Have_An_Account: '還未擁有帳號?',
+	Do_you_have_an_account: '是否擁有帳號?',
+	Do_you_have_a_certificate: 'Do you have a certificate?',
+	Do_you_really_want_to_key_this_room_question_mark: '您真的想要{{key}}這個房間嗎?',
+	E2E_How_It_Works_info1: 'You can now create encrypted private groups and direct messages. You may also change existing private groups or DMs to encrypted.',
+	E2E_How_It_Works_info2: 'This is *end to end encryption* so the key to encode/decode your messages and they will not be saved on the server. For that reason *you need to store this password somewhere safe* which you can access later if you may need.',
+	E2E_How_It_Works_info3: 'If you proceed, it will be auto generated an E2E password.',
+	E2E_How_It_Works_info4: 'You can also setup a new password for your encryption key any time from any browser you have entered the existing E2E password.',
+	edit: '編輯',
+	edited: '已編輯',
+	Edit: '編輯',
+	Edit_Status: '編輯狀態',
+	Edit_Invite: '編輯邀請',
+	End_to_end_encrypted_room: 'End to end encrypted room',
+	end_to_end_encryption: 'end to end encryption',
+	Email_Notification_Mode_All: 'Every Mention/DM',
+	Email_Notification_Mode_Disabled: '禁用',
+	Email_or_password_field_is_empty: '電子郵件或密碼字段為空',
+	Email: '電子郵件',
+	EMAIL: '電子郵件',
+	email: '電子郵件',
+	Empty_title: '空白標題',
+	Enable_Auto_Translate: '開啟自動翻譯',
+	Enable_notifications: '開啟訊息通知',
+	Encrypted: '已加密',
+	Encrypted_message: '加密訊息',
+	Enter_Your_E2E_Password: 'Enter Your E2E Password',
```

Comment:
@David-Tsui Can you translate these missing strings?

Reply:
Okay, give me some time~